src/components/LegislativeProcess/VotingProcessImageCircle/VotingProcessImageCircle.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	export let src = '';
+	export let alt = '';
+	export let title: string;
+	export let description = '';
+</script>
+
+<div class="flex-1 max-w-[280px]">
+	<div class="mb-2 flex justify-center text-center">
+		{#if src}
+			<img class="w-32 h-32 rounded-full" {src} alt={alt ? `${alt}` : `${title}`} />
```

Comment:
For `alt`, we can just write

```svelte
<img alt={alt || title} />
```

Both alt and title is a string, no need to put them inside the `${}`. Then we can use OR `||`, if alt is falsy (eg. undefined or empty) then `title` will be used.